cli/src/main.rs

```diff
@@ -875,6 +884,9 @@ impl Parse {
         let mut stats = parse::Stats::default();
 
         for path in &paths {
+            if parse_stdin {
+                println!(); // place user input and parser output on separate lines
+            }
```

Comment:
Conceptually, I wouldn't think this should be in a loop, even though the length of `paths` is guaranteed to be one with the current code - it'd make more sense for this to happen outside the loop, unless we handle "multiple" stdin files